snowfall/common.py

```diff
@@ -75,22 +81,20 @@ def load_checkpoint(filename: Pathlike, model: AcousticModel) -> Tuple[int, floa
     return epoch, learning_rate, objf
 
 
-def save_checkpoint(
-        filename: Pathlike,
-        model: AcousticModel,
-        epoch: int,
-        learning_rate: float,
-        objf: float,
-        local_rank: int = 0
-) -> None:
+def save_checkpoint(filename: Pathlike,
```

Comment:
There is a lot of formatting changes that obfuscate what you actually did. I guess we need common auto-formatting script or settings. Preferably something that can work easily with IDEs like PyCharm (not sure what you use).

Reply:
Sorry for the inconvenience.

I am using vim configured with `yapf==0.27.0` for auto-formatting. The configure file for yapf is from
https://github.com/k2-fsa/k2/blob/master/.style.yapf

What is the formatter that you are using in PyCharm?

Reply:
I'm using the built-in formatter, but PyCharm can be configured to use something external (e.g. I used https://github.com/psf/black in the past for one project).

Reply:
I can switch to `black` in vim. Is there any config file for black? Which version are you using?

Reply:
For config check this out: https://github.com/psf/black#configuration-format 

I am not using any particular version at the moment, I guess any one is fine (we should freeze it in the requirements).